File: 1672-richest-customer-wealth/1672-richest-customer-wealth.cpp
class Solution {
public:
    int maximumWealth(vector<vector<int>>& accounts) {
        int maxWealth = 0;
        
        for(int i=0;i<accounts.size();i++){
            int sum=0;
            for(auto money: accounts[i]){
                sum+=money;
            }
            maxWealth = max(maxWealth, sum);
        }
        return maxWealth;
    }
};

File: 1672-richest-customer-wealth/1672-richest-customer-wealth.java
class Solution {
    public int maximumWealth(int[][] accounts) {
        int maxWealth = 0;
        
        for(int[] account: accounts){
            int sum = 0;
            for(int money: account){
                sum += money;
            }
            maxWealth = Math.max(maxWealth, sum);
        }
        
        return maxWealth;
    }
}